doc/CODING.md

```diff
@@ -51,33 +51,56 @@ intent to add a new rule here.
 
 See [DESIGN](DESIGN.md) document too, they complement each other.
 
+### Coding Style ###
 
-### C Guidelines ###
+- Limits
 
  * Functions should not exceed 100 lines.
  * Files should not exceed 1000 lines.
- * A line should not be longer than 72 characters.
- * Split up when those limits are reached.
- * Rationale: Readability with split windows.
- * The compiler shall not emit any warning (or error).
+ * A line should not be longer than 80 characters.
```

Comment:
What about 120? Do you really have a workflow/screen/editor where 120 characters are problematic?